components/utilities/HtmlTooltip.tsx

```diff
@@ -0,0 +1,13 @@
+import { styled, Tooltip, tooltipClasses, TooltipProps } from '@mui/material'
+
+export const HtmlTooltip = styled(({ className, ...props }: TooltipProps) => (
+  <Tooltip {...props} classes={{ popper: className }} />
+))(({ theme }) => ({
+  [`& .${tooltipClasses.tooltip}`]: {
+    backgroundColor: '#f5f5f9',
```

Comment:
Can we change this background color to somewhat dark? Just to make it consistent

Reply:
Very true...made changes